### PR TITLE
Update for services 1.0

### PR DESCRIPTION
**Pull request by ian-r-rose**

### CODE REVIEW

File: src/gapi.ts [hunk 1]
```diff
@@ -301,18 +301,9 @@ function getCurrentUserProfile(): gapi.auth2.BasicProfile {
...
-    responseText: message
-  };
-  return {
-    event: undefined,
-    xhr: xhr as XMLHttpRequest,
-    ajaxSettings: null,
-    throwError: xhr.responseText,
-    message: xhr.responseText
-  } as any as ServerConnection.IError;
+function makeError(code: number, message: string): ServerConnection.ResponseError {
```
Comment (blink1073) [inline src/gapi.ts]: Comment above references IError

Comment (ian-r-rose) [reply src/gapi.ts]: :+1:

**Comment by blink1073** [comment]
Note: I plan to use `prettier` in the main repo, which will disable formatting checks in tslint, then we run it as an npm script to format our code, similar to the integrity check.

**Comment by blink1073** [comment]
https://alexjoverm.github.io/2017/06/12/Use-Prettier-with-TSLint-and-be-happy/

**Comment by ian-r-rose** [comment]
Sounds good.

**Comment by ian-r-rose** [comment]
Can we distribute globally runnable scripts in `@jupyterlab/buildutils` such that extension authors can do formatting or integrity checks?

**Comment by blink1073** [comment]
Yup, some of that is already done.

**Pull request merged**